scripts/cooptimizer_pipeline/step1_prices.jl

```diff
@@ -63,7 +63,7 @@ transform_single_time_series!(sys_rts_da, horizon_DA, interval_DA)
 #interval_RT = Minute(5)
 #horizon_RT = 24
 interval_RT = Hour(1)
-horizon_RT = 12 * 24
+horizon_RT = 12
```

Comment:
We probably need to document better that in these cases it makes more sense to limit the window size of the lower level problem